lib/MusicBrainz/Server/Controller/Role/EditRelationships.pm

```diff
@@ -449,9 +452,11 @@ role {
         while (my ($key, $relationship_order) = each %reordered_relationships) {
             my ($link_type_id) = split /-/, $key;
 
+            my $link_type = $link_types_by_id->{$link_type_id};
+
             push @edits, $self->reorder_relationships(
                 $c, $form,
-                link_type_id => $link_type_id,
+                link_type => $link_type,
```

Comment:
Why is this better? I don't have a problem with the change, but maybe explain the reasoning in the commit message?

Reply:
Added a commit message here.  (P.S. You left this comment twice so I deleted one.  :))